WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -128,6 +130,18 @@ class MainActivity : AppCompatActivity() {
         val jsObjName = "jsObject"
         val allowedOriginRules = setOf<String>("https://gcoleman799.github.io")
 
+        // Check if the system is set to light or dark mode
+        val nightModeFlag = resources.configuration.uiMode.and(Configuration.UI_MODE_NIGHT_MASK)
```

Comment:
nit: I think you can do: `val nightModeFlag = resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK`